pkgs/tools/misc/blsd/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, buildGoPackage, fetchFromGitHub, pkgconfig, libgit2 }:
+
+buildGoPackage rec {
+  name = "blsd-${version}";
+  version = "unstable-2017-07-27";
```

Comment:
The version begins with a digit, so unstable is not part of the version, but the suffix of the name. (And for packages which are unlikely to ever have a version number it is fine to drop it.)